### ISSUE TITLE
incomplete data using webservice getUsers

### ISSUE THREAD

**Issue by jolypas** [vitalidze/traccar-web#1140]
hello
I'm using restfull APi  getUsers service without problem but I'm not getting all data expected from the "User" structure 
For example email address is not returned 

What I got for each user is :

  "id" : 47,
  "login" : "fanny",
  "admin" : false,
  "manager" : true,
  "userSettings" : {
    "id" : 53,
    "speedUnit" : "kilometersPerHour",
    "timePrintInterval" : 10,
    "followedDeviceZoomLevel" : 16,
    "zoomLevel" : 11,
    "centerLongitude" : 2.256793975834742,
    "centerLatitude" : 48.91398145739631,
    "mapType" : "GOOGLE_NORMAL",
    "maximizeOverviewMap" : false
  },
  "readOnly" : false,
  "archive" : true,
  "expired" : false,
  "passwordHashMethod" : "MD5",

Any reason not to get all data ?

Thanks

**Comment by vitalidze** [comment]
That's correct. Not all fields from [User](https://github.com/vitalidze/traccar-web/blob/dev/src/main/java/org/traccar/web/shared/model/User.java) model are exposed through the API. There were no need to expose all of them. To show a field it must have getter/setter and there should be no `@JsonIgnore` annotation. You can easily change the `User` class and expose needed fields.
